api/app/signals/apps/email_integrations/tests/test_actions.py

```diff
@@ -532,6 +569,16 @@ def test_send_email_for_parent_signals(self):
         self.assertEqual(len(mail.outbox), 0)
         self.assertEqual(Note.objects.count(), 0)
 
+    @override_settings(FEATURE_FLAGS={
+        'REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT': False,
```

Comment:
See comment on `settings/base.py`. Rename feature flag to `REPORTER_MAIL_CONTACT_FEEDBACK_ALLOWS_CONTACT_ENABLED`